meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/ReportFormat.java

```diff
@@ -0,0 +1,33 @@
+package edu.neu.ccs.prl.meringue;
+
+import org.jacoco.report.*;
```

Comment:
Using `import ....*` is considered bad in Java, but I see some other file has it.

Reply:
Yes... I'll expand this into separate imports.